SDPi_Supplement/asciidoc/sdpi-supplement-issues.adoc

```diff
@@ -1,4 +1,19 @@
-= Issues and Questions
+
+// = Open Issues and Questions
 
 NOTE:  This file will contain both the open and closed issues section relating to the supplement.  Linked to Github Issues.
 
+[sdpi_offset=clear]
+== Open Issues and Questions
```

Comment:
Does it really make sense to foster a list of open and closed issues in the document itself rather than just pointing to the github issues page (maybe with an enabled filter for SDPi supplement). This is yet another chapter that needs extra care which is otherwise being automated by github.

Reply:
Not my call ... it is part of the IHE TF supplement template ... for-ever!   Once we get the structure integrated into master, I was going to pick off one of the ToI that is resolved and update it into the document.  Bottom line, though is that we need:
a) An entry with a simple description & link to the Github Issue
b) Github issue that includes subtasks for each of the document updates that need to be made (and tracked)
c) Github issue link to the Confluence ToI for that item where we document the discussion, etc.